cssom-view-1/Overview.bs

```diff
@@ -1596,11 +1597,11 @@ partial interface HTMLImageElement {
 };
 </pre>
 
-The <dfn attribute for=HTMLImageElement>x</dfn> attribute, on getting, must return the x-coordinate of the left <a>border edge</a> of the
+The <dfn attribute for=HTMLImageElement>x</dfn> attribute, on getting, must return the [=scaled=] x-coordinate of the left <a>border edge</a> of the
 first [=CSS/box=] associated with the element, relative to the <a>initial containing block</a> origin, ignoring any
 <a>transforms</a> that apply to the element and its ancestors, or zero if there is no [=CSS/box=].
```

Comment:
Is this the only case where we ignore transforms but not zoom out of curiosity? (Seems fine but...)

Reply:
Yes it is, to my knowledge. Unfortunate, but obscure and doesn't matter much.